test/test_mps.py

```diff
@@ -7745,17 +7743,7 @@ def get_samples():
                 cpu_out = op(*cpu_args, **cpu_kwargs)
                 mps_out = op(*mps_args, **mps_kwargs)
 
-                if op.name == "nn.functional.conv2d" and dtype == torch.float32:
-                    atol = 1e-4
```

Comment:
We don't need these tolerate values anymore ?

Reply:
I moved them over into the function metadata in [torch/testing/_internal/common_methods_invocations.py](https://github.com/pytorch/pytorch/pull/87529/files#diff-18a0c3e7cd07089b34760028c387d39de1aa393172f1d5f123692106b3a3d144)

At least I think I did, and tests look happy.